widget_tweaks/tests.py

```diff
@@ -162,6 +162,8 @@ def test_silence_without_field(self):
         self.assertEqual(res, "")
         res = render_field("nothing", 'add_class', 'some')
         self.assertEqual(res, "")
+        res = render_field("nothing", 'remove_attr', 'some')
```

Comment:
It's best not to mix quote styles; go either with double or single quotes.

Reply:
Mix quote style has been used in the tests file. I merely copied it. Would you like me to make those changes across the file?

Reply:
That should be done in a separate PR. New code should, as a rule of thumb, use modern standards and good style, unless they make it inconsistent and unreadable as compared to old code. Places like these are good spots to start fixing the styling and making it consistent :) Either style is really fine.